sys/auto_init/netif/auto_init_rail.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Hochschule RheinMain
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+
+#ifdef  MODULE_RAIL
+
+#include <stdint.h>
+
+#define ENABLE_DEBUG  (0)
+#include "debug.h"
+
+#include "log.h"
+#include "net/gnrc/netif/ieee802154.h"
+#include "net/gnrc.h"
+
+#include "rail_drv.h"
+#include "rail_params.h"
+
+#define RAIL_MAC_STACKSIZE      (THREAD_STACKSIZE_DEFAULT)
+#ifndef RAIL_MAC_PRIO
+#define RAIL_MAC_PRIO           (GNRC_NETIF_PRIO)
+#endif
+
+
+// there could be more interfaces, if BLE and 802.15.4 are used together, or the
+// 2.4 GHz and the 868 MHz transceiver?
+
+#define RAIL_IFACE_NUM 1 // todo calc number?
```

Comment:
Not used?

Reply:
Not yet, but I guess I can remove this stuff for now. 
The rail lib supports multible protocols running parallel, like BLE and 802.15.4. And it should be possible (I think/hope) to use both frequencies in parallel. Than we need support for multible interfaces for the same driver.